app/views/articles/_full_comment_area.html.erb

```diff
@@ -1,9 +1,12 @@
 <% cache("whole-comment-area-#{@article.id}-#{@article.last_comment_at}-#{@article.show_comments}", expires_in: 2.hours) do %>
   <div id="comments" data-updated-at="<%= Time.current %>">
     <% if @article.show_comments %>
-      <div class="comments-container-container">
+        <div class="relative flex justify-between items-center mx-2 s:mx-7 mb-2">
```

Comment:
Looks like the _start_ of the wrapping div was removed but the end was not here, that's causing the funky thing below. Also causing an issue with the elements being too close to bottom of box.

<img width="860" alt="Screen Shot 2020-04-30 at 7 33 21 PM" src="https://user-images.githubusercontent.com/3102842/80768827-76b15d80-8b19-11ea-8796-0cdbc655fd0f.png">

Reply:
> Looks like the _start_ of the wrapping div was removed but the end was not here, that's causing the funky thing below. Also causing an issue with the elements being too close to bottom of box.

I figured it was something silly. I’ve been starting at the coffee for too long. 👀